calva/repl-window.ts

```diff
@@ -169,6 +169,10 @@ class REPLWindow {
         })
         try {
             this.postMessage({ type: "repl-response", value: await this.evaluation.value, ns: this.ns = ns || this.evaluation.ns || this.ns });
+            if(this.evaluation.ns && this.ns != this.evaluation.ns) {
+                // the evaluation changed the namespace so set the new namespace.
+                this.setNamespace(this.evaluation.ns);
```

Comment:
Awesome. I have been figuring about this a bit, but obviously not enough to figure this fix out. Thanks!